easycukes-scm/src/main/java/com/worldline/easycukes/scm/utils/GitHelper.java

```diff
@@ -107,15 +115,165 @@ public static void commitAndPush(@NonNull File directory, String username,
             final PersonIdent author = new PersonIdent(username, "");
             git.commit().setCommitter(author).setMessage(message)
                     .setAuthor(author).call();
-            log.info("Commited the changes in the Git repository...");
+            log.info(COMMIT_MESSAGE);
             // and finally push
             final UsernamePasswordCredentialsProvider userCredential = new UsernamePasswordCredentialsProvider(
                     username, password);
             git.push().setCredentialsProvider(userCredential).call();
-            log.info("Pushed the changes in remote Git repository...");
+            log.info(PUSH_MESSAGE);
         } catch (final GitAPIException e) {
             log.error(e.getMessage(), e);
             throw e;
         }
     }
+    
+    /**
+	 * Create a new branch in the local git repository
+	 * (git checkout -b branchname) and finally pushes new branch on the remote repository (git push)
+	 *
+	 * @param directory the directory in which the local git repository is located
+	 * @param username  the username to be used while pushing
+	 * @param password  the password matching with the provided username to be used
+	 *                  for authentication
+	 * @param message   the commit message to be used	 
+	 */
+	public static void createBranch(@NonNull File directory, String branchName, String username,
+									String password, String message) throws GitAPIException {
+
+		try {
+			final Git git = Git.open(directory);
+
+			final UsernamePasswordCredentialsProvider userCredential = new UsernamePasswordCredentialsProvider(
+					username, password);
+
+			CreateBranchCommand branchCommand = git.branchCreate();
+			branchCommand.setName(branchName);
+			branchCommand.call();
+			log.info("Develop branch created");
```

Comment:
Sorry It was removed earlier, While reverting style it got injected. 

I logged it for testing while creation of first branch. I will remove that log